src/dev/precommit_hook/get_files_for_commit.js

```diff
@@ -21,29 +21,24 @@ import { File } from '../file';
  */
 export async function getFilesForCommit(gitRef) {
   const simpleGit = new SimpleGit(REPO_ROOT);
-  const gitRefForDiff = gitRef ? gitRef : '--cached';
-  const output = await fcb((cb) => simpleGit.diff(['--name-status', gitRefForDiff], cb));
+  const gitCatPrefix = ':' + gitRef ? gitRef + ':' : '';
```

Comment:
I assume you want this here? 
```suggestion
  const gitCatPrefix = ':' + (gitRef ? gitRef + ':' : '');
```

Reply:
You are right, though I was 100% sure it is needed and don't know how it could possibly work.